src/IPatchworkProtocol.sol

```diff
@@ -258,6 +268,15 @@ interface IPatchworkProtocol {
         bool active; /// true for present
     }
 
+    /**
```

Comment:
Not part of this change, but is it more consistent to define ProtoclFeeOverride as

struct ProtocolFeeOverride {
    ProtocolFeeConfig config;
    bool active;
}

Also mentioned elsewhere, but I think it makes more sense to name these generically as FeeConfig/Override in all places where these are being used for both scope and protocol fees.